ip/draft-ietf-detnet-ip.xml

```diff
@@ -111,22 +111,28 @@
       <t>
         This document specifies the DetNet data plane operation for IP hosts
         and routers that provide DetNet service to IP encapsulated data.  No
-        DetNet specific encapsulation is defined to support IP flows, instead
+        DetNet-specific encapsulation is defined to support IP flows, instead
         the existing IP and higher layer protocol header information is used to
         support flow identification and DetNet service delivery. Common data plane
         procedures and control information for all DetNet data planes
         can be found in the <xref target="I-D.ietf-detnet-data-plane-framework"/>.
       </t>
       <t>
         The DetNet Architecture models the DetNet related data plane functions
-        decomposed into two sub-layers: functions into two sub-layers: a
+        as two sub-layers: functions into two sub-layers: a
         service sub-layer and a forwarding sub-layer.  The service sub-layer is
-        used to provide DetNet service protection and reordering. The
+        used to provide DetNet service protection (e.g., by packet replication 
+		and packet elimination functions) and reordering. The
         forwarding sub-layer is used to provides congestion protection (low
-        loss, assured latency, and limited out-of-order delivery). Since no DetNet
-        specific headers are added to support DetNet IP flows, only the
-        forwarding sub-layer functions are supported using the DetNet IP
-        defined by this document. Service protection can be provided on a per
+        loss, assured latency, and limited out-of-order delivery). 
+		The service sub-layer requires additional fields to provide its service; 
+		for example to provide service protection via PREOF, each packet must 
+		be assigned a sequence number. Since no DetNet-specific fields 
+		(such as sequence number) are added to support DetNet IP flows, 
+		the DetNet service sub-layer for IP cannot support PREOF. Thus only 
```

Comment:
These PREOF is a detail that doesn't belong in the IP draft -- e.g., if someone defines a different protection mechanism it shouldn't impact this doc! How about:
The service sub-layer generally requires additional fields to provide its service; for example  see <xref target="I-D.ietf-detnet-data-plane-framework"/>. Since no DetNet-specific fields 
are added to support DetNet IP flows, only

Reply:
Yes, text is intended here for explanation, why no PREOF for IP. Is it disturbing? Do You propose to delete? If yes, which part?

Reply:
As mentioned on the list, this document shouldn't depend on a reliability approach used in other documents or need to change when new reliability mechanisms are introduced.  I'm proposing:
OLD:
		The service sub-layer requires additional fields to provide its service; 
		for example to provide service protection via PREOF, each packet must 
		be assigned a sequence number. Since no DetNet-specific fields 
		(such as sequence number) are added to support DetNet IP flows, 
		the DetNet service sub-layer for IP cannot support PREOF. Thus only 
NEW
The service sub-layer generally requires additional fields to provide its service; for example see [DETNET-MPLS]. Since no DetNet-specific fields are added to support DetNet IP flows, only

Reply:
OK. I have updated this.